src/DynamoPackages/Package.cs

```diff
@@ -92,6 +92,9 @@ public bool TypesVisibleInManager
         public IEnumerable<string> Keywords { get { return _keywords; } set { _keywords = value; RaisePropertyChanged("Keywords"); } }
 
         private IEnumerable<string> hostDependencies = new List<string>();
+        /// <summary>
+        /// Package Host Dependencies
```

Comment:
can you describe this a bit more - this is an API that will likely be encountered by an external user. What does being in this list imply?